controllers/dependencies.go

```diff
@@ -0,0 +1,168 @@
+package controllers
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+
+	"os"
+
+	"path/filepath"
+
+	git "github.com/go-git/go-git/v5"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	REPO_CLONE_PATH         = "/tmp/operator-pipelines/"
+	PIPELINE_MANIFESTS_PATH = "ansible/roles/operator-pipeline/templates/openshift/pipelines"
+	TASKS_MANIFESTS_PATH    = "ansible/roles/operator-pipeline/templates/openshift/tasks"
+)
+
+func (r *OperatorPipelineReconciler) reconcilePipelineDependencies(meta metav1.ObjectMeta) error {
+
+	// Cloning operator-pipelines project to retrieve pipelines and tasks
+	// yaml manifests that need to be applied beforehand
+	// ref: https://github.com/redhat-openshift-ecosystem/certification-releases/blob/main/4.9/ga/ci-pipeline.md#step-6---install-the-certification-pipeline-and-dependencies-into-the-cluster
+
+	repo := "https://github.com/redhat-openshift-ecosystem/operator-pipelines.git"
```

Comment:
Could this be a constant as well?

Reply:
Fair enough. My bad. I think I passed too fast my eyes on that. I just changed it.